getting_started/step_by_step/your_first_game.rst

```diff
@@ -926,6 +926,9 @@ StartButton
 
 -  Text: ``Start``
 
+To make the text of the ``MessageLabel`` fit the game's window click
```

Comment:
I suggest rewording it slightly (for better grammar):

```
To make the text of the ``MessageLabel`` fit the game window, click
the small icon next to the Text property. This will open a new window
where you can add line breaks between words.
```